### ISSUE TITLE
AdopteUnMec custom filtering

### ISSUE THREAD

**Issue by Amustache** [OpenTermsArchive/dating-declarations#88]
Re: #63, after looking into the code, I noticed that the whole document is availble in `cgvuDatas["cookiesDatas"]["data"]` within a js script. Is it possible ti select that way or not?

**Comment by MattiSG** [comment]
If I understand correctly, not at the moment. But please have a look at https://github.com/ambanum/OpenTermsArchive/issues/745, and if it matches your situation, add this document as an example! 🙂

**Comment by Amustache** [comment]
It's not _exactly_ JSON, but it can be parsed as such... But yes, the situation is quite the same.

Should I reference it anyway?

**Comment by MattiSG** [comment]
Aah sorry, I had not understood. I then assume that `cgvuDatas` is a JS object. If it is populated through an XHR (in which case there is an underlying JSON resource), then yes, please reference it!
If it is loaded as part of a JS script, then I don't think it is relevant to use it as a source: unless the HTML document is especially hard to parse, the fragility brought by parsing code variables seems too risky for Open Terms Archive.
